### PR TITLE
Adds support for adding instance methods as handlers

### PR DESCRIPTION
**Pull request by ReneHollander**
Adding an instance method to flask and then running flask injector would cause a crash as reported in #34.

### CODE REVIEW

File: flask_injector.py [hunk 1]
```diff
@@ -37,10 +38,20 @@
 T = TypeVar('T', LocalProxy, Callable)
 
 
+def instance_method_wrapper(im):
```
Comment (jstasiak) [inline flask_injector.py]: This can be typed like `wrap_fun` below (T -> T)

File: flask_injector_tests.py [hunk 2]
```diff
@@ -364,6 +364,33 @@ def do_nothing():
...
+        def from_injected_service(self, service: HelloWorldService):
+            return service.get_value()
+        
+        def static_value(self):
+            return "test message 2"
+
+    app = Flask(__name__)
+    hello_world = HelloWorld()
+    app.add_url_rule('/from_injected_service', 'from_injected_service', hello_world.from_injected_service)
+    app.add_url_rule('/static_value', 'static_value', hello_world.static_value)
```
Comment (jstasiak) [inline flask_injector_tests.py]: I don't think we require this – `from_injected_service` should test everything we need?

**Comment by jstasiak** [comment]
The newly added code needs to have type hints added and we're good to go.

File: flask_injector.py (hunk 1 above)
Comment (ReneHollander) [reply flask_injector.py]: Done.

File: flask_injector_tests.py (hunk 2 above)
Comment (ReneHollander) [reply flask_injector_tests.py]: When trying it out I had no issues as long as there was nothing being injected, so I assumed it was working. Due to this I also added a test for it.

**Comment by ReneHollander** [comment]
Done and Thanks!

**Comment by jstasiak** [comment]
I'll deal with those build failures, don't worry about those.

**Comment by jstasiak** [comment]
Thanks for the patch!

**Comment by ReneHollander** [comment]
Thank you for merging and fixing the typing issue again...

**Comment by jstasiak** [comment]
No worries. This has just been released as part of 0.12.0.

**Pull request merged**